web/js/app.js

```diff
@@ -26,16 +26,15 @@ import ErrorBoundary from './containers/error-boundary';
 import Debug from './components/util/debug';
 
 // Dependency CSS
-import '../../node_modules/bootstrap/dist/css/bootstrap.css';
-import '../../node_modules/jquery-ui-bundle/jquery-ui.structure.css';
-import '../../node_modules/jquery-ui-bundle/jquery-ui.theme.css';
+import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import '../../node_modules/jquery-ui-bundle/jquery-ui.structure.min.css';
+import '../../node_modules/jquery-ui-bundle/jquery-ui.theme.min.css';
 import '../../node_modules/icheck/skins/square/grey.css';
```

Comment:
I wonder if we can also remove these `icheck` css imports, as well as remove this dependency from the `package.json`?  At a glance it doesn't look like we are using this anywhere anymore.

Reply:
That would be good to remove. I'll check that out.